.github/workflows/helm.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: helm/kind-action@a1b0e391336a6ee6713a0583f8c6240d70863de3 # v1.12.0
         if: steps.list-changed.outputs.changed == 'true'
         with:
-          node_image: kindest/node:v1.31.4
+          node_image: kindest/node:v1.32.0
```

Comment:
We must use a sha256 sum as in https://github.com/kubernetes-sigs/kind/releases/tag/v0.29.0
> NOTE: You _must_ use the `@sha256` digest to guarantee an image built for this release, until such a time as we switch to a different tagging scheme. Even then we will highly encourage digest pinning for security and reproducibility reasons.